packages/sites/src/pages/update-page.ts

```diff
@@ -23,9 +23,18 @@ import { removeUnusedResources } from "../layout/remove-unused-resources";
  */
 export function updatePage(
   model: IModel,
-  patchList: string[],
-  requestOptions: IHubRequestOptions
+  maybePatchList: string[] | Record<string, any>,
+  maybeRequestOptions?: IHubRequestOptions
 ): Promise<IUpdateItemResponse> {
+  let patchList = maybePatchList as string[];
+  let requestOptions = maybeRequestOptions;
+
+  // support old call signature
+  if (!Array.isArray(maybePatchList)) {
+    patchList = getWithDefault(maybePatchList, "patchList", []) as string[];
```

Comment:
the second arg is required so why not just use `maybePatchList.patchList || []` here?